docker/keystone/keystone-base/Dockerfile.j2

```diff
@@ -97,11 +97,12 @@ RUN sed -i -r 's,^(Listen 80),#\1,' /etc/httpd/conf/httpd.conf \
         {% set keystone_base_packages = [
             'apache2',
             'libapache2-mod-auth-mellon',
-            'libapache2-mod-auth-openidc',
             'libapache2-mod-wsgi-py3',
             'python3-ldappool'
         ] %}
 {{ macros.install_packages(keystone_base_packages | customizable("packages")) }}
+{% set mod_auth_openidc_releases = 'https://github.com/zmartzone/mod_auth_openidc/releases/download' %}
+RUN apt-get install ./{{ mod_auth_openidc_releases}}/v2.4.4.1/libapache2-mod-auth-openidc_2.4.4.1-1.bionic+1_amd64.deb
```

Comment:
Is that `./` in front valid?

Reply:
I'll know once the container builds + runs, but probably not, I don't know apt's / dpkg's behavior with urls just yet.